content/blog/scale-kubernetes-clusters-using-hpe-greenlake-terraform-provider.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Upgrade Kubernetes Clusters using HPE GreenLake Terraform Provider
+date: 2023-05-12T04:07:48.888Z
+author: Tanushi Agarwal
+authorimage: /img/photoforblog.jpg
+disable: false
+---
```

Comment:
Please add Tags here.
At the bottom of the CMS editor, you will see a section to add tags, "One at a time".
So, please add the tags you listed in the TAGS section at the end of your post, Make sure to add one tag at a time - you will have to click "Add Tags+" to add one tag. Repeat the same for each tag.
 And then remove the section "Tags" and its contents at the end of post.